src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.html

```diff
@@ -62,7 +62,7 @@
               i18n>At least one pool is required.</span>
       </div>
 
-      <cd-submit-button i18n
+      <cd-submit-button i18n [disabled] = 'generateBtnDisable'
```

Comment:
Fix alignment and favor double quotes over single ones
```suggestion
      <cd-submit-button i18n 
                        [disabled] = "generateBtnDisable"
```